Sources/StreamChatUI/CommonViews/InputTextView/InputTextView.swift

```diff
@@ -127,6 +130,13 @@ open class InputTextView: UITextView, AppearanceProvider {
     @objc open func handleTextChange() {
         placeholderLabel.isHidden = !text.isEmpty
         setTextViewHeight()
+
+        // If the user pasted text from the clipboard, we want to scroll to the caret position.
+        if textChangedFromClipboard, let selectedTextRange = self.selectedTextRange {
```

Comment:
If the text that's being copied is from another app, wouldn't the `selectedTextRange` being `nil`?

Reply:
What do you mean? 🤔 From another app? The caret position is automatically set to the end of the pasted text, no matter where you pasted it from